tests/resolve_imports/tests.cmake

```diff
@@ -0,0 +1,25 @@
+
+# Set the test name, 'test_' will be prepended to the
+# name set here
+set(CURRENT_TEST resolve_imports)
+# Set a category name to enable running commands like:
+#    ctest -R <category-label>
+# which will run the tests matching this category-label.
+# Can be left empty (or just not set)
+set(${CURRENT_TEST}_CATEGORY io)
+list(APPEND LIBCELLML_TESTS ${CURRENT_TEST})
+# Using absolute path relative to this file
+set(${CURRENT_TEST}_SRCS
+  ${CMAKE_CURRENT_LIST_DIR}/file_parser.cpp
+)
+#set(${CURRENT_TEST}_HDRS
+#  ${CMAKE_CURRENT_LIST_DIR}/<test_header_files.h>
+#)
+
+
+set(CELLML_SINE_MODEL_RESOURCE "${CMAKE_CURRENT_SOURCE_DIR}/resources/sine_approximations.xml")
```

Comment:
We shouldn't be doing this every time, we should have a common resources script that we include from the tests/CMakeLists.txt file.

Reply:
addressed by #172.

Reply:
Will be resolved once #172 is done.